### PR TITLE
Add solarized theme

### PR DESCRIPTION
**Pull request by timja**

### CODE REVIEW

**Comment by timja** [comment]
@daniel-beck it would be good to add the appropriate topic to solarized so that it shows up with the right category in plugin manager

File: README.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ First you will want to install a theme plugin through the Jenkins Update Center.
 
 * [Dark theme](https://github.com/jenkinsci/dark-theme-plugin)
 * [Dark theme (OS settings respected)](https://github.com/jenkinsci/dark-theme-plugin)
+* [Solarized theme](https://plugins.jenkins.io/solarized-theme/)
```
Comment (daniel-beck) [inline README.md:28]: If Dark Theme plugin has two list entries (one for each theme it implements), Solarized needs three: Light, Dark, and System.

Comment (timja) [reply README.md:28]: it could prob just be one for dark

File: README.md [hunk 2]
```diff
@@ -25,6 +25,7 @@ First you will want to install a theme plugin through the Jenkins Update Center.
 
 * [Dark theme](https://github.com/jenkinsci/dark-theme-plugin)
 * [Dark theme (OS settings respected)](https://github.com/jenkinsci/dark-theme-plugin)
```
Comment (timja) [inline README.md:27]:
```suggestion
```

**Review by daniel-beck — APPROVED** [review]
Ideally would be replaced by a dynamic list, but with just two plugins so far I doubt we're there yet.

https://github.com/search?q=topic%3Atheme+org%3Ajenkinsci+fork%3Atrue is a bit messy, and https://www.jenkins.io/doc/developer/extensions/ looks outdated/broken 😦

**Pull request merged**